dev-cpp/fbthrift/fbthrift-2025.04.14.00-r1.ebuild

```diff
@@ -40,8 +40,12 @@ DEPEND="
 	dev-libs/xxhash
 	sys-libs/zlib:=
 "
+DEPEND="
+	${COMMON_DEPEND}
+	~dev-cpp/mvfst-${PV}:=
```

Comment:
Hm, if the files _installed by_ Thrift require it, it's also an RDEPEND?

Reply:
Thrift never needs it at runtime, it's only required at build time by Thrift's dependencies. Does that still count as an RDEPEND?

Reply:
Yes.